### PR TITLE
Install application on every workspace in the account instead of a single workspace

### LINKED ISSUE #255: Support for Multiple Workspcaes on AWS and Azure
Many of our customers have multiple workspaces (s0me of them in thousands). This is required to run per workspace. What is the guidance on running this for multiple workspaces.

### PR DESCRIPTION
**Pull request by nfx**
Fixes #255

### CODE REVIEW

**Comment by FastLee** [comment]
Can you provide more details on how it works?
How to setup the install to install for the account rather then the workspace?
I would like to run some tests.

**Comment by zpappa** [comment]
Where is the documentation? I don't see a MD file or a comment - just code.

**Comment by zpappa** [comment]
Can you please lead with documentation?

It would be great to understand how this is proposed to work.

Account Admins are rarely involved in upgrading to UC, this feature makes assumptions about who will run the upgrade, i.e. the account admin.

**Comment by nfx** [comment]
@zpappa How else we'll automatically update 2000 workspaces if there's no account admin access? It's a prerequisite

File: src/databricks/labs/ucx/config.py [hunk 1]
```diff
@@ -45,6 +45,8 @@ class ConnectConfig:
     azure_client_secret: str | None = None
     azure_environment: str | None = None
     cluster_id: str | None = None
+    username: str | None = None
+    password: str | None = None
```
[the remark this answers is not in the document]
Comment (nfx) [reply src/databricks/labs/ucx/config.py:49]: 🤷🏻‍♂️ we don't even encrypt it in ~/.databrickscfg

**Comment by zpappa** [comment]
I'm pointing out that the premise is inaccurate. 

No one will be able to update 2000 workspaces at a time. No one will have access to 2000 workspaces at a time. This is not how the product is used in the real world.

We should talk more about this tomorrow. ☝️

**Comment by pohlposition** [comment]
We've seen large organizations pause their upgrade because it's not clear who the rightful account admin should be.  In some cases, we grant them the ability to create multiple UC metastores (different admins) within the same cloud region.  After our "UC as Default" work, there will no longer need to be a Metastore Admin.

**Comment by nfx** [comment]
out of date

**Pull request closed without merging**